README.md

```diff
@@ -48,11 +48,23 @@ Run your project (Cmd+R)
 
 ### Installation (Android)
 
-* Add Gradle configuration changes
+* In `android/setting.gradle`
 
-Run `react-native link react-native-device-info` in your project root.
```

Comment:
Not sure you're modifying the correct part here.  Using `react-native link` should work.

I thought you were going to remove the directions for `rnpm`